data/rao-result/rao-result-impl/src/main/java/com/farao_community/farao/data/rao_result_impl/PstRangeActionResult.java

```diff
@@ -18,41 +18,37 @@
  */
 public class PstRangeActionResult extends RangeActionResult {
 
-    private Integer preOptimTap;
+    private int initialTap;
+    private Integer postPraTap;
     private final Map<State, Integer> tapPerState;
 
     public PstRangeActionResult() {
         super();
-        preOptimTap = null;
         tapPerState = new HashMap<>();
     }
 
-    public int getPreOptimTap() {
-        return preOptimTap;
-    }
-
     public int getPreOptimizedTapOnState(State state) {
```

Comment:
In the case where state is curative, and there was an auto RA on the PST, wouldn't we want to return a postAraTap rather the postPraTap?

Reply:
Nevermind, I just noticed this "// does only handle RA applicable in PREVENTIVE and CURATIVE instant"